tests/main_app/jobs_workers/test_fix_nested_main_files_worker.py

```
@@ -232,3 +233,314 @@ def process_fix_nested_side_effect(filename, user, cancel_event=None):
+
+        assert result["success"] is False
+        assert result.get("cancelled") is True
+        assert "Cancelled" in result["message"]
+
+    def test_cancellation_after_fix(self):
+        """Test cancellation after fix but before verify."""
+        cancel_event = threading.Event()
+
+        call_count = 0
+
+        def set_cancel_after_detect(*args, **kwargs):
```

Ruff (0.15.2)

[warning] Unused function argument: `args`

(ARG001)